essay.tex

```diff
@@ -0,0 +1,61 @@
+ 
+The research from various authors shows that the key to productive teams is: to make sure you build a sense of community, to make sure that the team are involved in work they find interesting and to let them experience other types of management so they can decide which is best.
+
+\section{Agile in the Gaming Industry}
+
+The adaptive nature of Agile means that it's well suited for software development. Some of the reflective work from software development industry its self \cite{Practitioners}, leads credence to the above research. However when looking within the gaming industry not all aspect of Agile can be considered the best option \cite{RyseCrisis} a reflective look at Agile development mainly on Scrum revealed that Agile was very useful at the initial stages of development, but as time progressed the need for a more structured or linear approach was necessary to ensure that deadlines were met. Though later expands on this point to say that being able to change to a different style of development could also be considered Agile in it's own way. 
+
+There are many useful studies which have taken place for both the software and management side of using Agile, however the gaming industry combines several disciplines which need to co-operate. I have been unable to find much in the way of quantified research upon the topic of motivation and morale within a game development company and the effects Agile has had upon them. I believe a further study into the subject matter would be prudent, especially considering the huge implications of motivation on a teams productivity.
+
+
+
+\section{Conclusion}
+
+In conclusion using the information from within this paper, it's safe to assume that Agile as a management system can be effective in software and games development. It's also important to note that there has been little research on management styles and their effects on motivation within the games industry exclusively. Though it is a topic that has been quite deeply explored in other works. \cite {EnhancePerformance,SoftwareProcess,LargeAgile,MicrosoftExp,UsingAgile,AgileDevelopment,MotivationTeamwork} Many of these works have excellent research into the subject matter however they also focus on other factors to motivation as proposed by Asproni.
+
+Overall the it would appear that Agile is the ultimate methodology, however as proposed by Patrick Payne not all situations are suitable for the Agile methodology. He proposed that assessing your situation is an important part of making a successful project, and that Agile is about being adaptive. The example he gave was ``I wouldn't use a hammer to cut a piece of paper.''  The research I have done has often spoken about Agile as the only method and the greatest thing, it's important however to also recognize that every group is different and people need to have different team setting in order to reach their full potential
```

Comment:
Missing a reference and page number. You *must* get referencing right to score marks on this assignment! It is, arguably, the most important criterion after *analysis*.